content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/critical-cve-policy.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Ensure there are no High or Critical CVEs in your workloads
+"
+type: "article"
+description: "Understanding Identity and Access Management in Chainguard Enforce"
+date: 2023-04-12T15:22:20+01:00
+lastmod: 2023-04-12T15:22:20+01:00
+draft: false
+images: []
+tags: ["policy-controller", "Policies", "Enforce", "Product"]
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 820
+toc: true
+---
+
+While Common Vulnerabilities and Exposures (CVEs) are undesirable at any time, the security standards of some industries strictly regulate the allowance of _high_ or _critical_ CVEs in industry-specific software.  For example, in the payment industry, the [PCI Security Standards Council](https://www.pcisecuritystandards.org/) requires that all vulnerabilities with a Common Vulnerability Scoring System (CVSS) score higher than 4 are addressed.
+
+For engineers and security professionals working in these contexts, it’s essential to know if images have high or critical CVEs before deploying them. But tracking these CVEs manually can be difficult, especially when regularly pulling or updating large numbers of images for your workloads.
```

Comment:
I thought having a "container" in there may help with people finding this resource

```suggestion
For engineers and security professionals working in these contexts, it’s essential to know if container images have high or critical CVEs before deploying them. But tracking these CVEs manually can be difficult, especially when regularly pulling or updating large numbers of images for your workloads.
```